docs/content/v2025.1/explore/transactions/transactional-ddl.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Transactional DDL
+headerTitle: Transactional DDL
+linkTitle: Transactional DDL
+description: Learn how YugabyteDB handles DDLs in a transaction
+headcontent: Learn how YugabyteDB handles DDLs in a transaction
+tags:
+   feature: tech-preview
+menu:
+  v2025.1:
+    identifier: transactional-ddl
+    parent: explore-transactions
+    weight: 300
+type: docs
+---
+
+Data Definition Language (DDL) statements modify the structure of your data. PostgreSQL allows executing such statements within a transactional block and supports rolling them back as part of a transaction rollback. For two transactions undergoing conflicting DDLs, it also provides isolation guarantees by taking appropriate object locks during DDL processing.
+
+YugabyteDB's transactional DDL provides similar guarantees for rolling back DDL operations done inside a transaction block. However, because object locks are still under active development, isolation guarantees are weaker than in PostgreSQL. See [Limitations](#limitations) for more information.
+
+## Enable transactional DDL
+
+This feature is {{<tags/feature/tp>}}. Support for transactional DDL is disabled by default, and to enable the feature, set the [yb-tserver](../../../reference/configuration/yb-tserver/) flag `yb_ddl_transaction_block_enabled` to true.
```

Comment:
It's a parameter though, not a flag right?

Reply:
Its a gflag.